specification/search/data-plane/Microsoft.Azure.Search.Data/preview/2019-05-06-preview/searchindex.json

```diff
@@ -857,6 +902,24 @@
       "additionalProperties": true,
       "description": "A single bucket of a facet query result. Reports the number of documents with a field value falling within a particular range or having a particular value or interval."
     },
+    "SearchFeatures": {
+      "properties": {
+        "uniqueTokenMatches": {
+          "type": "number",
+          "readOnly": true,
+          "format": "double",
+          "description": "The number of unique tokens from the search query that matched this field."
+        },
+        "similarityScore": {
+          "type": "number",
+          "readOnly": true,
+          "format": "double",
+          "description": "The similarity score computed between the search query and this field."
+        }
+      },
+      "additionalProperties": true,
```

Comment:
I don't think you need `additionalProperties` here since this shouldn't be modeled as a dictionary.

Reply:
Also, for any properties that will always be present in the response, please add them to a `required` property array (there are other examples of this elsewhere in the specs). This helps AutoRest to generate non-nullable references where applicable.